README.md

```diff
@@ -150,17 +161,23 @@ _(All commands are launched from the root directory of your node project.)_
 
 A catalog file is a json file, representing a single language.
 
+The first time you need to create the folder for the locales you want, for example:
```

Comment:
"Before the first run you'll need to create one folder per needed locale, for instance:"